### ISSUE TITLE
Java Akka documentation should use Abstract actor 

### ISSUE THREAD

**Issue by ktoso** [playframework/playframework#7544]
Docs on https://www.playframework.com/documentation/2.6.x/JavaAkka (the java ones) should use `AbstractActor` it's nicer to use from Java and the current recommended go-to API for untyped Actors in Java.

An Actor now looks like:

```java
import akka.actor.AbstractActor;
import akka.event.Logging;
import akka.event.LoggingAdapter;

public class MyActor extends AbstractActor {
  private final LoggingAdapter log = Logging.getLogger(getContext().getSystem(), this);

  @Override
  public Receive createReceive() {
    return receiveBuilder()
      .match(String.class, s -> {
        log.info("Received String message: {}", s);
      })
      .matchAny(o -> log.info("received unknown message"))
      .build();
  }
}
```

**Comment by ktoso** [comment]
I think I'll just fix those for you :-)

**Comment by ktoso** [comment]
Why closed? PR was not merged yet

**Comment by cchantep** [comment]
there is no need to keep open 2 tickets

**Comment by ktoso** [comment]
You treat PRs as tickets in Play? Ok, sure whatever your workflow is... Was rather confusing that ticket closed without resolving thing being merged first hm

**Comment by gmethvin** [comment]
@cchantep The correct way to have an issue and use "Fixes #xxx" on the PR for that issue. For a while we were using the feature to convert issues into pull requests, but we're not doing that anymore since that feature is deprecated and it was never used consistently.

**Comment by richdougherty** [comment]
We do sometimes just create a PR without having an associated issue, but I think it's OK to have both an issue and a PR too.

**Comment by richdougherty** [comment]
Fixed by #7599.
